App/Sources/DesignSystem/View/MainMissionScrollView.swift

```diff
@@ -0,0 +1,46 @@
+import SwiftUI
+
+struct MainMissionScrollView: View {
+    var tester: String
+    var subtitle: String
+    var price: String
+    
+    var body: some View {
+        VStack {
+            ScrollView(.horizontal, showsIndicators: false) {
+                HStack(spacing: 16) {
+                    ForEach(1..<7) { _ in
```

Comment:
```suggestion
                    ForEach(1..<7) { _ in
```
어차피 서버에서 리스트 받아서 사용해야 하니까 이렇게 1..<7 하는 것보다 배열 상수 하나 생성해서 ForEach문에 사용하면 좋을 것 같아요~

Reply:
알겠습니다 지금 바로 넣어서 올리도록 하겠습니다!!